modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -51,17 +42,18 @@
 
 	subtle_message = span_emote("<b>[user]</b>[space]<i>[user.say_emphasis(subtle_message)]</i>")
 
-	for(var/mob/ghosts in GLOB.dead_mob_list)
-		if(!ghosts.client || isnewplayer(ghosts))
+	var/list/viewers = get_hearers_in_view(1, user)
+
+	for(var/mob/ghost in GLOB.dead_mob_list)
+		if(!ghost.client || isnewplayer(ghost))
 			continue
```

Comment:
```suggestion
```
We check for client right afterward and `dead_mob_list` already excludes `/mob/dead/new_player`

Reply:
Old code that I forgot to remove, woops!